lib/SMTP/SmtpClientFactory.php

```diff
@@ -58,7 +58,7 @@ public function __construct(IConfig $config,
 	 *
 	 * @return Horde_Mail_Transport
 	 */
-	public function create(Account $account): Horde_Mail_Transport {
+	public function create(Account $account): Horde_Mail_Transport_Smtphorde {
```

Comment:
```suggestion
	public function create(Account $account): Horde_Mail_Transport {
```

See https://github.com/nextcloud/mail/blob/14a7362f6152e0d79831be1dc1ae69a146ddc59f/lib/SMTP/SmtpClientFactory.php#L65